utils/grpcutils/grpc.go

```diff
@@ -36,7 +36,7 @@ func X509Certificate(privKey crypto.PrivateKey) (*tls.Certificate, error) {
 
 	// extract the TLSConfig from it which will contain the generated x509 certificate
 	// (ignore the public key that is returned - it is the public key of the private key used to generate the ID)
-	libp2pTlsConfig, _ := id.ConfigForAny()
+	libp2pTlsConfig, _ := id.ConfigForPeer("")
```

Comment:
I thought `ConfigForAny` did the same thing as ConfigForPeer("") , no?

Reply:
it did. they dropped it in the latest release